src/BPGates.jl

```diff
@@ -409,6 +419,7 @@ const h = tHadamard
 const p = tPhase
 const hp = h*p
 const ph = p*h
+# TODO: What is this????
```

Comment:
the 6 gates that are "good permutation gates", i.e. the ones from `good_perm_tuple`, i.e. `GoodSingleQubitPerm`, but represented as Clifford tableaux, for more general-purpose simulations. Used in the `toQCcircuit` conversion utility.